sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalog.scala

```diff
@@ -261,6 +261,20 @@ trait ExternalCatalog {
       table: String,
       partialSpec: Option[TablePartitionSpec] = None): Seq[CatalogTablePartition]
 
+  /**
+   * List the metadata of all partitions that belong to the specified table
```

Comment:
s/List/Lists

Reply:
Prefer to keep same with other comments.